src/helm/benchmark/scenarios/wmt_14_scenario.py

```diff
@@ -0,0 +1,95 @@
+from typing import List, Any
+from datasets import load_dataset
+from helm.common.hierarchical_logger import hlog
+from .scenario import Scenario, Instance, Reference, TRAIN_SPLIT, VALID_SPLIT, TEST_SPLIT, CORRECT_TAG, Input, Output
+
+MAX_TRAIN_INSTANCES = 20_000  # This is arbitrary, but 20,000 training examples should be enough.
+
+
+class WMT14Scenario(Scenario):
+    """
+    The 2014 Workshop on Statistical Machine Translation:
+    https://aclanthology.org/W14-3302.pdf
+
+    The scenario consists of 5 subsets, each of which is a parallel corpus between English and another language. The
+    non-English languages include Czech, German, French, Hindi, and Russian.
+
+    For each language pair, the validation and test set each includes around 3,000 examples, while the training set is
+    usually much larger. We therefore randomly downsample the training set to speedup data processing.
+
+    Task prompt structure:
+
+        Translate {source_language} to {target_language}:
+        {Hypothesis} = {Reference}
+
+    Example from WMT14 Fr-En:
+
+        Hypothesis: Assemblée générale
+        Reference: General Assembly
+    """
+
+    name = "WMT_14"
+    description = "Scenario for the 2014 Workshop on Statistical Machine Translation"
+    tags = ["machine_translation"]
+
+    def __init__(self, source_language, target_language):
+        super().__init__()
+        valid_languages = set(["cs", "de", "fr", "hi", "ru", "en"])
+        self.source_language = source_language
+        self.target_language = target_language
+        # The source/target langauges must be included in WMT14
+        assert self.source_language in valid_languages and self.target_language in valid_languages
+        # The source language and the target language should be different
+        assert self.source_language != self.target_language
+        # One of the languages should be English
+        assert self.source_language == "en" or self.target_language == "en"
```

Comment:
Suggestion: Generally, `assert` should be used to check internal invariants, rather than user inputs. I'd suggest doing `raise ValueError` instead with a useful error message.

Reply:
Sure!